### ISSUE TITLE
404 Link behind Share the App (flutter).

### ISSUE THREAD

**Issue by RaiAnsar** [WorldHealthOrganization/app#446]
**This One**
![image](https://user-images.githubusercontent.com/35567562/77776909-8a782880-7070-11ea-98d7-fedf9d49d473.png)
**Text and Link you get after clicking the above button** -> `Check out the official COVID-19 Guide App https://www.who.int/covid-19-app`

**404 Page on WHO Website.**
![image](https://user-images.githubusercontent.com/35567562/77776444-d8d8f780-706f-11ea-8821-834fcd6f58b6.png)

**Comment by epicfaace** [comment]
Duplicate of #291.

**Comment by epicfaace** [comment]
Actually, I guess it's not a duplicate. We should make sure this site is populated before release.

**Comment by ZacharyEilers** [comment]
Is that page included in the repo? I would assume not. For now, we could just have it point to the WHO's official coronavirus page to avoid being blocked on the app store. In a later update, we can have the link point directly to the App Store/Google Play Store link.

**Comment by kassim** [comment]
perhaps something like [Firebase Dynamic Links](https://firebase.google.com/docs/dynamic-links) would help (redirects to Google Play or iTunes or a website depending on the context)

**Comment by ZacharyEilers** [comment]
I don't think firebase dynamic links would solve this problem. Seems like unnecessary added complexity. It would be much easier to just check the platform and link directly to the app store link

**Comment by hspinks** [comment]
Closing in favor of #291
